.betterer.results

```diff
@@ -4744,6 +4734,12 @@ exports[`better eslint`] = {
     "public/app/features/search/service/sql.ts:5381": [
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"]
     ],
+    "public/app/features/search/state/SearchState.ts:5381": [
+      [0, 0, 0, "Do not use any type assertions.", "0"],
+      [0, 0, 0, "Do not use any type assertions.", "1"],
+      [0, 0, 0, "Do not use any type assertions.", "2"],
+      [0, 0, 0, "Do not use any type assertions.", "3"]
+    ],
```

Comment:
don't understand these, there are no any types in this file

Reply:
they're flagging type assertions (`foo as Type`) instead of `any`. given it's just moving around existing code it's fine 🙂